packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/JSON/ComplexJSONRenderer.tsx

```diff
@@ -27,13 +27,18 @@ export function ComplexJSONRenderer(props: ComplexJSONRendererProps) {
     <>
       <Typography
         variant={'smallText1'}
-        sx={{ py: 0.5, color: 'grey.600', fontStyle: 'italic' }}
+        sx={{
+          py: 0.5,
+          color: 'grey.600',
+          fontStyle: 'italic',
+          cursor: 'pointer',
+        }}
+        onClick={() => setExpandAll(v => !v)}
```

Comment:
Minor improvement for this `ComplexJSONRenderer` where the "Expand all" text is now also interactive, not just the button next to the text.